README.md

```diff
@@ -73,6 +73,44 @@ The analysis will return: `http` (in try), `crypto`, `util` and `fs`.
 
 > ⚠️ There is also a lot of suspicious code example in the root cases directory. Feel free to try the tool on these files.
 
+## Warnings CONSTANTS
+
+This section describes how use warnings CONSTANTS.
+
+The structure of the Constants is as follows:
+```
+/**
+ * @property {object}  CONSTANTS                         - The default values for Constants.
+ * @property {object}  CONSTANTS.Warnings                - The default warnings.
+ * @property {object}  CONSTANTS.Warnings[name]          - The default warnings name (parsingError, unsafeImport etc...).
+ * @property {string}  CONSTANTS.Warnings[name].i18n     - i18n token.
+ * @property {string}  CONSTANTS.Warnings[name].code     - Used to perform unit tests.
+ */
+ 
+export const CONSTANTS = {
+  Warnings: Object.freeze({
+    parsingError: {
+      i18n: "sast_warnings.ast_error"
+      code: "ast-error",
+    },
+    ...otherWarnings
+  })
+};
+```
+The snippet below we use `js-x-ray` dependency through the package `NodeSecure/Scanner` for get the warnings.
+
+We make a call to `i18n` through the package `NodeSecure/i18n` to get the translation.
+```
+import * as scanner from "@nodesecure/scanner";
+import * as i18n from "@nodesecure/i18n";
+
+const payload = await scanner.from("packageName", { 4 }, initLogger("packageName"));
+
+if (payload.warnings.length) {
+  const value = i18n.getToken(payload.warnings[0].i18n);
+}
```

Comment:
```js
import * as jsxray from "@nodesecure/js-x-ray";
import * as i18n from "@nodesecure/i18n";

console.log(i18n.getToken(jsxray.CONSTANTS.Warnings.parsingError.i18n));
```

An example like this is better (payload.warnings is not about JS-X-Ray warnings).

Reply:
Yes